### ISSUE TITLE
docker/setup-buildx-action fail if DOCKER_TLS_VERIFY is true

### ISSUE THREAD

**Issue by Hazmi35** [Hazmi35/dind-actions-runner#1]
![image](https://user-images.githubusercontent.com/32807631/182023754-5ad25c92-db69-4baa-8ba1-ddbaa7a388b2.png)

Also reproducible by entering the worker container: Running `$ docker buildx create` will result in error:
```sh
error: could not create a builder instance with TLS data loaded from environment. Please use `docker context create <context-name>` to create a context for current environment and then create a builder instance with `docker buildx create <context-name>`
```

This is can somewhat be fixed with: https://github.com/docker/setup-buildx-action/issues/16#issuecomment-687807834

but, then another problem appears:
![image](https://user-images.githubusercontent.com/32807631/182024375-95daf056-ab59-4b0a-9688-96df20023f93.png)

It is caused by buildx not using the `DOCKER_TLS_VERIFY=1`, *I'll find a real solution, and reporting this problem to buildx team, until then, I would disable TLS.*

**TL/DR:** `docker/setup-buildx-action` would fail if using `DOCKER_TLS_VERIFY` feature

**Please do not expose port `2376` on DinD container, and please be aware about [UFW+Docker problem](https://github.com/chaifeng/ufw-docker#problem)**
